### ISSUE TITLE
Line consisting only of dashes inside literal block breaks syntax highlighting

### ISSUE THREAD

**Issue by jeffreyscarpenter** [asciidoctor/asciidoctor-intellij-plugin#324]
I'm including output from Cassandra's shell (cqlsh) in my text inside a literal block. It appears as if the line in the cqlsh output that consists only dashes with no leading whitespace confuses the syntax highlighting.

```
==== Some Heading

[listing]
....
cqlsh:my_keyspace> UPDATE user SET emails = {
  'mary@example.com' } WHERE first_name = 'Mary';
cqlsh:my_keyspace> SELECT emails FROM user WHERE first_name =
  'Mary';

 emails
----------------------
 {'mary@example.com'}

(1 rows)
....

==== Next heading
```

Visual of the lack of highlighting on "Next heading":
![Screen Shot 2019-08-24 at 1 51 32 PM](https://user-images.githubusercontent.com/12115970/63642592-5110b200-c676-11e9-942a-74680930e4e9.png)

I can work around this by adding a leading space to the line of dashes. This is then technically not the exact output from the tool, but acceptable in my case.

Version info:
Asciidoc Plugin 0.26.15 
IntelliJ IDEA 2017.3.7 (Ultimate Edition)
Build #IU-173.4710.11, built on April 4, 2019

**Comment by ahus1** [comment]
Hi Jeffrey, the version 0.26.15 you are using didn't support the "...." delimiter. This causes the syntax highlighting to be out of sync.

TL;DR: Please update to the most recent version 0.29.10. 

0.26.18 was the first pre-release to support this delimiter,  0.28.2 was the first regular release to support it. Side note: I recommend upgrading to IntelliJ IDE 2019.x with JBR11 for fewer JavaFX bugs. On the other hand the AsciiDoc plugin 0.29.10 should be compatible with IntelliJ 2017.3 AFAIK.

**Comment by ahus1** [comment]
Closing due to timeout.
